crates/syntax/rust.ungram

```diff
@@ -438,7 +438,10 @@ FormatArgsExpr =
   ')'
 
 FormatArgsArg =
-  (Name '=')? Expr
+  arg_name:FormatArgsArgName? Expr
+
+FormatArgsArgName =
+  '=' // This also has a name, but it's any token and we can't put it here
```

Comment:
Let's put an `#ident` here, while still not completely correct its better than not encoding any token in the grammar here maybe do a named one like `format_args_arg_name:'#ident'` (if that works), then we can filter the name out in codegen to not generate the method)

Reply:
welp